### PR TITLE
add headful WPT for github actions

### PR DESCRIPTION
**Pull request by thiagowfx**
Re-created https://github.com/GoogleChromeLabs/chromium-bidi/pull/383

### CODE REVIEW

**Comment by thiagowfx** [comment]
There was no need to close https://github.com/GoogleChromeLabs/chromium-bidi/pull/383/. The issue was:

Wrong:

```
+        if: ${{ matrix.head == "wpt" }}
```

Right:

```
+        if: ${{ matrix.head == 'wpt' }}
```

(i.e. quotes)

**Comment by thiagowfx** [comment]
On my macOS laptop:

```
Ran 78 tests finished in 880.3 seconds.
  • 32 ran as expected. 0 tests skipped.
  • 10 tests timed out unexpectedly
  • 36 tests had unexpected subtest results

./wpt/wpt run --webdriver-binary ./runBiDiServer.sh   --manifest MANIFEST.jso  33.82s user 14.65s system 5% cpu 14:54.33 total
```

**Comment by thiagowfx** [comment]
@sadym-chromium could you help me update the expectations? There seems to be something wrong.

I have used:

```
% ./wpt/wpt update-expectations --product chromium --manifest MANIFEST.json --metadata ./wpt-headful-metadata/mapper out/wptreport-mapper.json
```

**Comment by sadym-chromium** [comment]
I updated the expectations, but I'd keep all of them under `wpt-metadata/${{runner}}/${{head-mode}}`.

**Comment by thiagowfx** [comment]
> I updated the expectations, but I'd keep all of them under wpt-metadata/${{runner}}/${{head-mode}}.

Thanks, will adjust it. How did you do it? Same command as I did?

**Comment by thiagowfx** [comment]
> I updated the expectations, but I'd keep all of them under wpt-metadata/${{runner}}/${{head-mode}}.

Done.

**Comment by thiagowfx** [comment]
The expectations still fail.

File: .github/workflows/wpt-mapper.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ on:
 jobs:
   wpt:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        head: [wpt, wpt-headful]
```
Comment (sadym-chromium) [inline .github/workflows/wpt-mapper.yml:16]: Consider [adding configurations](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations):
```
    strategy:
      matrix:
        include:
          - head: "headful"
            command_prefix: "xvfb-run --auto-servernum"
            wpt_argument: "--webdriver-arg="--headless=false"
          - head: "headless"
            command_prefix: ""
            wpt_argument: "--webdriver-arg="--headless=true"
```

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**